### DISCUSSION TITLE
from esri *.asc to export stl

### DISCUSSION THREAD

**Discussion by sugizo** [domlysz/BlenderGIS#902] [Q&A]
**trial 1**
1. generate esri *.asc file from saga gis
2. import esri *.asc file via blend gis
![Screenshot 2024-06-19 013028](https://github.com/domlysz/BlenderGIS/assets/2418295/0e1bac7b-da69-4e83-8fb0-ec34255df77c)

3. export *.stl from blender
4. result 23 mb (not expected)
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/28cfaa03-2080-4c86-8afa-ac6c14aba56e)

**trial 2**
1. generate esri *.asc file from saga gis
2. import esri *.asc file via blend gis
![Screenshot 2024-06-19 013028](https://github.com/domlysz/BlenderGIS/assets/2418295/0e1bac7b-da69-4e83-8fb0-ec34255df77c)

3. use mesh delaunay in blender gis
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/a9eafb85-6a63-471e-8386-c5064c720f5b)

4. export *.stl from blender
5. result 45 mb (not expected)
![image](https://github.com/domlysz/BlenderGIS/assets/2418295/392c5997-275f-4b1e-9c0f-61893486197f)

**question**
how to get correct elevation *.stl file using blender gis ?

best regards

**Comment by domlysz** [comment]
sound like the xy coordinates of your asc file are in degrees whereas altitude are in meters

**Comment by sugizo** [comment]
how to set xy coordinates of asc file in meters, 
so that can be used in blender gis ?

**p.s.**
already tried *.asc generated from gdal, grass and saga
have the same value
